controllers/api/post-routes.js

```diff
@@ -69,12 +64,12 @@ router.get('/:id', (req, res) => {
     });
 });
 
-router.post('/',   (req, res) => {
-  // expects {title: 'Taskmaster goes public!', post_url: 'https://taskmaster.com/press', user_id: 1}
+router.post('/', (req, res) => {
+  
   Post.create({
     title: req.body.title,
     post_content: req.body.post_content,
```

Comment:
I think this should match Post model:
user_id
title
description
start_date
end_date